documentation/book/ref-resource-limits-and-requests.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+// assembly-resource-limits-and-requests.adoc
+
+[id='ref-resource-limits-and-requests-{context}']
+= Resource limits and requests
+
+Resource limits and requests can be configured using the `resources` property in following resources:
+
+* `Kafka.spec.kafka`
+* `Kafka.spec.kafka.tlsSidecar`
+* `Kafka.spec.zookeeper`
+* `Kafka.spec.zookeeper.tlsSidecar`
+* `Kafka.spec.entityOperator.topicOperator`
+* `Kafka.spec.entityOperator.userOperator`
+* `Kafka.spec.entityOperator.tlsSidecar`
+* `KafkaConnect.spec`
+* `KafkaConnectS2I.spec`
+
+== Resource requests
+
+Requests specify the amount of resource which will be reserved for given container.
+Since the amount will be reserved it will be always available.
```

Comment:
Since the amount will be reserved it will be always available.

TO

Reserving the resources will ensure that they are always available.